ebpf_extensions/neteventebpfext/netevent_ebpf_ext_event.c

```diff
@@ -74,15 +74,16 @@ typedef struct netevent_ext_header
 typedef struct netevent_ext_function_addresses
 {
     netevent_ext_header_t header;
+    netevent_capture_type_t capture_type;
     uint32_t helper_function_count;
     uint64_t* helper_function_address;
 } netevent_ext_function_addresses_t;
 
 // Dispatch table for the client module's helper functions
 static const void* _ebpf_netevent_ext_helper_functions[] = {(void*)&_ebpf_netevent_push_event};
```

Comment:
_ebpf_netevent_push_event may run at DISPATCH_LEVEL IRQL, so using a pushlock may result in a deadlock. We will need to consider some other synchronization mechanism such as a spin lock